src/lib/sidebar/PipelineSchemeForm.svelte

```diff
@@ -23,21 +20,16 @@
   // synced to $gjSchemeCollection until finish() is called
   let pipeline = $gjSchemeCollection.schemes[scheme_reference].pipeline!;
 
-  // TODO Note below we make sure "other" isn't in this list, just to make TS happy
-  let fundingSources = [
-    "atf2",
-    "atf3",
-    "atf4",
-    "atf4e",
-    "crsts",
-    "luf",
-  ] as Array<keyof FundingSources>;
-
-  // Check for all required values
-  $: errorMessage =
-    pipeline.scheme_type && pipeline.status && pipeline.timescale
+  // Svelte doesn't see nested updates in BudgetForm and TimingForm, so use a
```

Comment:
An existing test failed, alerting me to this problem -- yay tests! The fine-grained reactivity for individual fields doesn't bubble up to the parent component, so this is the simplest way I could figure out to plumb updates through